src/labels/CustomLabels.labels

```diff
@@ -1605,6 +1605,14 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <shortDescription>Warning displayed on Customizable Rollups page when enhanced RDs are enabled</shortDescription>
         <value>&lt;b&gt;WARNING: You have Enhanced Recurring Donations enabled. If you disable Customizable Rollups, rollups to Recurring Donations will no longer function.&lt;/b&gt;</value>
     </labels>
+    <labels>
+        <fullName>RD2_ElevatePermissionRequired</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error when user does not have a permission to change an RD originated in Elevate</shortDescription>
+        <value>Recurring Donations from Elevate Payment Services cannot be modified directly in NPSP at this time.</value>
```

Comment:
@sam-knox , can you please review the message?

Reply:
@RadSFDeveloper - So in this case, if a user did want to make an update, they would need to make it through Elevate?

Reply:
If the integration is enabled, user must have permissions in NPSP to change the Elevate RD. An integration user for now will have Elevate permissions for now (until bi-directional update is implemented).
@force2b , feel free to provide more detail if above is not correct.

Reply:
Ok, thanks. Based on the ACs in the WI, how about this message:

"Recurring Donations from Elevate Payment Services can't be modified in NPSP except by the Elevate Payment Services Integration user."

@RadSFDeveloper @force2b

Reply:
The message has been updated.